frontend/src/components/Common/HashText/index.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+const HashText = () => {
+  const hash = process.env.REACT_APP_GIT_HASH;
+  return (
+    <p
+      style={{
+        color: '#f5f7f8',
+        position: 'absolute',
+        bottom: '0',
+        fontSize: '0.7rem',
+        left: '1rem',
+      }}
+    >
+      version hash:{' '}
```

Comment:
let's make the whole text clicable. If no hash is provided, simply link to the repo